page/src/components/Filters/Filters.jsx

```diff
@@ -39,15 +39,15 @@ const Filters = ({ query, callForPapers, closedCaptions, country, onChange, onCl
       </div>
 
       <div className='filtersItem'>
-        <label for='filter-closed-captions'>Closed Captions:</label>
+        <label htmlFor='filter-closed-captions'>Closed Captions:</label>
```

Comment:
this file changes was fixing some warnings

Reply:
ah ah I wanted to do the same thing, thx :)